### ISSUE TITLE
New bookmark not shown on homepage

### ISSUE THREAD

**Issue by madprog** [mcansky/mjolk#25]
I added a bookmark on 10.03.2011 (id=356) and it doesn't show up on homepage (http://www.mjolk.net/)

All bookmarks in this list are mcansky's.

**Comment by mcansky** [comment]
yes, the cache layer is fucked up, I'm currently working on moving from heroku to something else and use varnish ...
